script/git-clone-others.sh

```diff
@@ -52,9 +80,17 @@ for repository in `cat "${scriptDir}/repository-list.txt"` ; do
         echo -- repository ${repository} --
         echo -- ${gitUrlPrefix}${repository}.git -- ${repository} --
         if [ "x${additionalGitOptions}" != "x" ]; then
-            echo -- additional Git options: ${additionalGitOptions} --
+            echo -- additional Git options: ${additionalGitOptions[@]} --
+        fi
+        git clone ${additionalGitOptions[@]} ${gitUrlPrefix}${repository}.git ${repository}
+
+        if [ "$UPSTREAM" = true ]; then
+            upstreamGitUrlPrefix=`echo ${gitUrlPrefix} | sed 's|\(.*github\.com[:/]\).*|\1kiegroup/|'`
+            echo -- adding upstream remote "${upstreamGitUrlPrefix}${repository}.git"
+            cd ${repository}
+            git remote add upstream ${upstreamGitUrlPrefix}${repository}.git
+            cd ..
```

Comment:
Isn't by default added origin remote which is in this case the added upstream remote ? Why is this better than the default clone command?

Reply:
Well, the first thing you have to do manually is to clone `droolsjbpm-build-bootstrap` so you have this script. After you run it, it will extract the part before `droolsjbpm-build-bootstrap` from the origin remote so this means e.g. in case of my fork `git@github.com:MarianMacik` since origin is my fork, upstream is a kiegroup original repository. But this doesn't matter anyway because all repositories will have only one remote specified, which will be either kiegroup or forked one. But if I want to create revapi PRs, I have to have both the fork and kiegroup remotes specified as I used them to pull the latest changes from upstream and then I commit a new PR on a separate branch which is then pushed to my forks.

So this new option will enable to clone repositories with both remotes specified -> so I can create PRs.

Reply:
ok, I got it as you will clone first your fork for droolsjbpm-build-bootstrap and clone other repos, but then you would add upstream remotes if used that new option. So this changes the order of how it was scripted originally. 
As you cloned all origin repos and add forks only to repos you are contributing.
Could you add this into the README or into the script help?

Reply:
well, I can do it also the other way around if you want, but the creation of the second remote wasn't automated until now :)
OK, will update the help.

Reply:
I agree, but just to keep clarity and not to confuse someone who already used it the opposite way is I think important.

But I agree your change make it more usable of the script ;)

Reply:
`--help` parameter added. I took inspiration from the `mvn-all.sh` script.